tests/python/unittest/test_gluon_model_zoo.py

```diff
@@ -35,7 +35,7 @@ def test_models():
     pretrained_to_test = set(['squeezenet1.1'])
 
     for model_name in all_models:
-        test_pretrain = True #model_name in pretrained_to_test
+        test_pretrain = model_name in pretrained_to_test
```

Comment:
I though you deliberately required all models to be tested.

Reply:
I did want to enable more tests here, since it didn't take too long on my machine. The problem is that sometimes download could timeout and cause failure, so I'd like to revert it to keep minimum for now.